lighthouse-core/config/seo.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  extends: 'lighthouse:default',
+  groups: {
+    'seo-mobile': {
+      title: 'Mobile Friendly',
+      description: 'Having a mobile-friendly website is a critical part of your online presence. In many countries, smartphone traffic now exceeds desktop traffic. '
+    },
+    'seo-content': {
+      title: 'Content Best Practices',
+      description: 'Ensure your markup enables crawlers to better understand your app\'s content.'
```

Comment:
Yeah this group is about expressing what the contents of the page are (title, description, etc), and the next group is about the crawler's ability to index the page and its links (X-Robots, nofollow, etc). The text is still a WIP, mostly just a placeholder for now while the audits are developed.